### PR TITLE
[#63] Move flutter project to `.template` folder

### PR DESCRIPTION
**Pull request by manh-t**
https://github.com/nimblehq/flutter_templates/issues/63

## What happened 👀

Our flutter_templates repository is a mixin between the template (contains Makefile, python scripts, etc., to initialize a Flutter project) and a Flutter templated app. It may confuse other devs when looking at the repository.

## Insight 📝

- Move Flutter project to `.template` directory.
- Update README.md on `root` directory to remove Flutter project parts
- Update README.md on `.template/README.md` to remove init the template parts.
- We don't have CI now. It will be implemented in #64 

## Proof Of Work 📹

App is still buildable.

### CODE REVIEW

File: .template/ios/Flutter/AppFrameworkInfo.plist [hunk 1]
```diff
@@ -21,6 +21,6 @@
   <key>CFBundleVersion</key>
   <string>1.0</string>
   <key>MinimumOSVersion</key>
-  <string>8.0</string>
+  <string>9.0</string>
```
Comment (luongvo) [inline .template/ios/Flutter/AppFrameworkInfo.plist:24]: Why do we upgrade the version here? 🤔

**Review by luongvo — COMMENTED** [review]
rest lgtm

File: .template/ios/Flutter/AppFrameworkInfo.plist (hunk 1 above)
Comment (luongvo) [reply .template/ios/Flutter/AppFrameworkInfo.plist:24]: @manh-t @doannimble From this PR https://github.com/nimblehq/flutter_templates/pull/71/files, I think we should update this version to `10.0` as well. We might don't need this to run the app locally but release https://docs.flutter.dev/deployment/ios#updating-the-apps-deployment-version

Comment (manh-t) [reply .template/ios/Flutter/AppFrameworkInfo.plist:24]: @luongvo When I run the app on iOS device, it auto-upgrades the version. Yeah, I will upgrade to `10.0` to match the requirement 🙏

Comment (luongvo) [reply .template/ios/Flutter/AppFrameworkInfo.plist:24]: What we did in the last project as well 

image

**Review by luongvo — APPROVED** [review]

**Review by doannimble — APPROVED** [review]
Good to go 🏃

**Pull request merged**